### PR TITLE
BCDA-3771 updated navigation and added page metadata

### PR DESCRIPTION
**Pull request by PatKelsh**
### Fixes [BCDA-3771](https://jira.cms.gov/browse/BCDA-3771)

Updated the global navigation for the header with the new pages for the static site glowup

### Proposed Changes

* Updated the jekyll configuration "global_nav" to include the new pages
* Added metadata tags to new pages

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Start locally and navigate to `localhost:4000` to see new header navigation links. Those links should all point to the correct pages

### CODE REVIEW

**Review by jeffcorpuz — APPROVED** [review]
🍏

File: _config.yml [hunk 1]
```diff
@@ -27,12 +27,12 @@ cssurl: /assets/css/static-main.css
 global_nav:
```
Comment (mtrang1263) [inline _config.yml:27]: Do we need entries for the quickstart and terms of service?

Comment (mtrang1263) [reply _config.yml:27]: Wasn't referenced in the associated FIGMA design.

**Review by mtrang1263 — APPROVED** [review]
LGTM

**Pull request merged**